### PR TITLE
Bump gnu-sed to 4.4

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/text/sed/test/results-64.master [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+SKIP: test-wcrtomb-w32-5.sh
+PASS: test-wctype-h
+PASS: test-xalloc-die.sh
+# TOTAL: 130
+# PASS:  116
+# SKIP:  13
+# XFAIL: 0
+# FAIL:  1
+# XPASS: 0
+# ERROR: 0
```
Comment (pyhalov) [inline components/text/sed/test/results-64.master]: Tests depend on locale/fr-extra , developer/debug/valgrind (and perhaps locale/fr). Better add them to REQUIRED_PACKAGES with corresponding comments.

**Comment by alarcher** [comment]
Fixed.

File: components/text/sed/Makefile [hunk 2]
```diff
@@ -20,37 +20,66 @@
...
-test:		$(TEST_32)
+test:		$(TEST_64)
 
-BUILD_PKG_DEPENDENCIES =	$(BUILD_TOOLS)
+# Test dependencies
+REQUIRED_PACKAGES += developer/debug/valgrind
+REQUIRED_PACKAGES += locale/de
+REQUIRED_PACKAGES += locale/en
+REQUIRED_PACKAGES += locale/fr
+REQUIRED_PACKAGES += locale/fr-extra
```
Comment (pyhalov) [inline components/text/sed/Makefile]: and developer/versioning/cvs (changes tests output)

**Comment by alarcher** [comment]
Fixed.

File: components/text/sed/Makefile [hunk 3]
```diff
@@ -20,37 +20,67 @@
...
+# Copyright (c) 2017, Aurelien Larcher.
 #
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		sed
-COMPONENT_VERSION=	4.2.2
+COMPONENT_VERSION=	4.4
+COMPONENT_FMRI=	text/gnu-sed
+COMPONENT_SUMMARY=	GNU implementation of sed, the Unix stream editor
```
Comment (pyhalov) [inline components/text/sed/Makefile:32]: Missing COMPONENT_CLASSIFICATION

Comment (alarcher) [reply components/text/sed/Makefile:32]: Fixed.

File: components/text/sed/test/results-64.master [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+PASS: test-locale
+PASS: test-localeconv
+FAIL: test-localename
+PASS: test-lseek.sh
+PASS: test-lstat
+PASS: test-malloca
+PASS: test-mbrtowc1.sh
+PASS: test-mbrtowc2.sh
+SKIP: test-mbrtowc3.sh
+PASS: test-mbrtowc4.sh
```
Comment (pyhalov) [inline components/text/sed/test/results-64.master:191]: This test depends on locale/zh_cn-extra

Comment (alarcher) [reply components/text/sed/test/results-64.master:191]: Fixed.

**Comment by alarcher** [comment]
@pyhalov Thanks for your help and thorough review :)

**Pull request merged**